root/io/event/Makefile

```diff
@@ -18,7 +18,7 @@ include $(ROOTTEST_HOME)/scripts/Event.mk
 
 #.SUFFIXES: .cxx .o .so
 
-CXXOPT:=-Z7
+CXXOPT:=-Z7 -std=c++11
```

Comment:
This sounds wrong.  What about c++14 and C++17 builds?

Reply:
The best place to update is scripts/Rules.mk updating the value of CXXFLAGS such that it has the same standard as the ROOT build.